platform/ios/src/MGLMapView.mm

```diff
@@ -1709,27 +1720,134 @@ - (void)handleRotateGesture:(UIRotationGestureRecognizer *)rotate
 {
     if ( ! self.isRotateEnabled) return;
 
+    if ([[NSUserDefaults standardUserDefaults] objectForKey:MGLRotationThresholdWhileZoomingKey]) {
```

Comment:
This check might be better done where the `UIRotationGestureRecognizer` is setup in the first place - i.e. change the selector.

Reply:
We might want to keep it in here in case the rotation threshold changes after the `commonInit`.